docs/standard/getting-started.md

```diff
@@ -1,18 +1,36 @@
 ---
 title: Getting Started
```

Comment:
should have keywords in the title. Getting Started with what? Getting Started with .NET perhaps?